src/components/Articles/Articles.vue

```diff
@@ -5,7 +5,10 @@
       <label class="define-article-title">Articles</label>
     </div>
 
-    <div class="section-container">
+    <div
+      class="section-container"
+      v-if="getBusinessInformation.hasRestrictions"
```

Comment:
What is this page (Articles) supposed to display if there are no provisions? ATM it looks like just a title and empty content. Is that correct, or should this page not display at all?

Reply:
For now it will be like that but there will be another subcomponent Resolution or Court Orders that is not done yet:

![image](https://user-images.githubusercontent.com/78114138/110831918-96dc9d80-824f-11eb-9b85-595989b7b67f.png)

Reply:
OK, perfect.